Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -707,6 +707,10 @@
 #define ENABLE_UNIFIED_PDF_DATA_DETECTION 1
 #endif
 
+#if !defined(ENABLE_UNIFIED_PDF_PAINT_SELECTIONS) && ENABLE(UNIFIED_PDF) && HAVE(PDFSELECTION_ENUMERATE_RECTS_AND_TRANSFORMS)
+#define ENABLE_UNIFIED_PDF_PAINT_SELECTIONS 1
```

Comment:
maybe UNIFIED_PDF_SELECTION_LAYER?

Reply:
Renamed in https://github.com/WebKit/WebKit/commit/ab434ecd8a237eb5256d62fbc8744be2db5813fc